ci/jenkins/testkomodo-libres-ctest.sh

```diff
@@ -76,45 +67,17 @@ copy_test_files () {
 }
 
 install_test_dependencies () {
-    pip install -r dev-requirements.txt
+
 }
```

Comment:
Remove entirely?

Reply:
Or I see why this is done now, but maybe add a comment that this is to avoid the default behavior of installing the python dependencies?

Reply:
sure :)